packages/extras/src/lib/hooks/useGltf.ts

```diff
@@ -58,13 +65,16 @@ export function useGltf<
   const loader = useLoader(GLTFLoader, {
     extend(loader) {
       if (opts?.useDraco) {
-        const dracoDecoderPath =
-          typeof opts.useDraco === 'string'
-            ? opts.useDraco
-            : 'https://www.gstatic.com/draco/versioned/decoders/1.4.3/'
-
-        const dracoLoader = new DRACOLoader().setDecoderPath(dracoDecoderPath)
-        loader.setDRACOLoader(dracoLoader)
+        if (typeof opts.useDraco === 'string' || typeof opts.useDraco === 'boolean') {
+          // default draco
+          const dracoLoader = useDefaultDracoLoader(
+            typeof opts.useDraco === 'string' ? opts.useDraco : undefined
+          )
```

Comment:
If above comment is implemented, this can be simplified to
```ts
const path = typeof opts.useDraco === 'string' ? opts.useDraco : 'https://www.gstatic.com/draco/versioned/decoders/1.4.3/'
if (!defaultDracoLoaderInstances[path]) {
  defaultDracoLoaderInstances[path] = new DRACOLoader().setDecoderPath(path)
}
loader.setDRACOLoader(defaultDracoLoaderInstances[path])
```